File: Problem6.py


def main(limit):

    sum_of_squares = sum(x**2 for x in range(limit+1))

    square_of_sum = sum(x for x in range(limit+1))**2

    print(square_of_sum - sum_of_squares)




if __name__ == '__main__':
    main(100)

File: Problem6.java

class Problem6 {
    public static void main(String[] argv) {
        int limit = 100;

        int sum_of_squares = 0;
        int square_of_sum = 0;

        for(int i = 0; i <= limit; i++) {
            sum_of_squares += i * i; 
        }

        for(int j = 0; j <= limit; j++) {
            square_of_sum += j; 
        }

        square_of_sum *= square_of_sum;

        System.out.println(square_of_sum - sum_of_squares);

    }
}
